File: pt1-semestre/nivelamento/questao2.c
//  Faça um método que receba um array de inteiros e mostre na tela o maior e o menor elementos do array.
#include<stdio.h>
int maior(int vet[]){
  int maior = vet[0];
  for(int i = 1; i < 10; i++){
    if(vet[i] > maior) maior = vet[i];
  }
  return maior;
}

int menor(int vet[]){
  int menor = vet[0];
  for(int i = 1; i < 10; i++){
    if(vet[i] < menor) menor = vet[i];
  }
  return menor;
}

int main(){
  int vet[10];
  for(int i = 0; i < 10; i++){
    scanf("%d", &vet[i]);
  }
  printf("Maior: %d\n", maior(vet));
  printf("Menor: %d\n", menor(vet));
}

File: pt1-semestre/nivelamento/questao2.java
//  Faça um método que receba um array de inteiros e mostre na tela o maior e o menor elementos do array.

import java.util.Scanner;

public class questao2 {
  public static int maior(int vet[]){
    int maior = vet[0];
    for(int i = 0; i < vet.length; i++){
      if(vet[i] > maior) maior = vet[i];
    }
    return maior;
  }

  public static int menor(int vet[]){
    int menor = vet[0];
    for(int i = 0; i < vet.length; i++){
      if(vet[i] < menor) menor = vet[i];
    }
    return menor;
  }

  public static void main(String[] args){
    Scanner sc = new Scanner(System.in);
    int vet[] = new int[10];

    for(int i = 0; i < vet.length; i++){
      vet[i] = sc.nextInt();
    }

    int maior = maior(vet);
    int menor = menor(vet);


    System.out.println("Maior: " + maior);
    System.out.println("Menor: " + menor);
  }
}